### PR TITLE
Fixed some issues

### LINKED ISSUE #68: Correct English names for value domains to match original paper
- Relationships (plural)
- Education/Career
- Recreation/Interests
- Mind, Body, & Spirituality
- Daily Responsibilities

### LINKED ISSUE #69: Rename app to "Behavioral Activation"
Please rename the app to Behavioral Activation, which is both in line with the original paper and in English US spelling

### LINKED ISSUE #71: Make English default language
When the app starts, Swedish is the default language. Please change to English.

### PR DESCRIPTION
**Pull request by malin-mallan**

### CODE REVIEW

**Comment by malin-mallan** [comment]
I tried på fix #69 as well but not sure if I did it right

File: src/language/LanguageProvider.tsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import React, { createContext, useContext } from 'react';
-import sv from 'language/sv';
+import en from 'language/en';
 
 import languages, { Language } from 'language';
 import Storage from 'storage';
 
-const LanguageContext = createContext(sv);
+const LanguageContext = createContext(en);
```
Comment (EMattfolk) [inline src/language/LanguageProvider.tsx:7]: This is not enough to change the default. You need to change the default setting in storage as well.

File: src/language/en.tsx [hunk 2]
```diff
@@ -81,7 +81,7 @@ const en = {
         ChooseEntryIconView: 'Choose icon',
         AddEntryView: 'Add activity',
         Information: 'Information',
-        BehaviouralActivation: 'Behaviour Activation',
+        BehaviouralActivation: 'Behavioral Activation',
```
Comment (EMattfolk) [inline src/language/en.tsx:84]: The name would also have to be changed in `app.json` . The repo should probably also be renamed.

**Review by EMattfolk — CHANGES_REQUESTED** [review]

File: app.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "expo": {
-    "name": "Behaviour Activation",
-    "slug": "BehaviourActivation",
+    "name": "Behavioral Activation",
+    "slug": "BehaviouralActivation",
```
Comment (EMattfolk) [inline app.json]:
```suggestion
    "slug": "BehavioralActivation",
```

**Review by EMattfolk — APPROVED** [review]

**Review by Neathan — APPROVED** [review]

**Pull request merged**